### DISCUSSION TITLE
ROS2 to CFbolt tutorial question

### DISCUSSION THREAD

**Discussion by kimseong-yeon** [bitcraze/discussions#272] [Q&A]
Hi. 
I'm connecting CF_bolt with ROS2 (on ubuntu 20.04)

I faced error message like below doing "ros2 launch crazyflie launch.py" 
at https://imrclab.github.io/crazyswarm2/usage.html#python-scripts

[crazyflie_server-4] terminate called after throwing an instance of 'std::runtime_error'
**[crazyflie_server-4]   what():  Issue with connection radiobroadcast://*/80/2M. Your radio with firmware 0.83 does not support broadcast communication. Please upgrade your Crazyradio firmware!**
[ERROR] [crazyflie_server-4]: process has died [pid 16894, exit code -6, cmd '/home/seongyeon/ros2_ws/install/crazyflie/lib/crazyflie/crazyflie_server --ros-args -r __node:=crazyflie_server --params-file /tmp/launch_params_uqm263hv'].

what means upgrading Crazyradio firmware?
does it mean **nrf51 firmware** or **crazyradio-firmware in github** or **crazyradio PA dongle firmware**?

my crazyradio firmware was 0.53 with below code

lsusb -d 1915:7777 -v | grep bcdDevice

**Comment by krichardsson** [comment]
Hi @kimseong-yeon !
I think you should ask this question in the [crazyswarm2 discussions](https://github.com/IMRCLab/crazyswarm2/discussions) instead as it is related to the Crazyswarm setup.

**Reply by kimseong-yeon** [reply]
thanks !
but i solved using ros1 tutorial

**Reply by krichardsson** [reply]
Great!
